pytorch_forecasting/models/temporal_fusion_transformer/tuning.py

```diff
@@ -23,6 +27,26 @@
 optuna_logger = logging.getLogger("optuna")
 
 
+# todo: Remove this class once lightning allows the pass of weights_only to tuner
```

Comment:
can you add a todo witih version number, so it gets checked with every minor release? E.g., todo next.minor.version.
Release managers typically search for version numbers of the release to be done.

Reply:
same for other todos